oqs-template/generate.yml

```diff
@@ -405,17 +405,98 @@ sigs:
         pretty_name: 'Dilithium2'
         oqs_meth: 'OQS_SIG_alg_dilithium_2'
         oid: '1.3.6.1.4.1.2.267.7.4.4'
-        code_point: '0xfea0'
+        code_point: '0xfe98'
```

Comment:
This is impossible: How can OID remain the same but code point change? Please explain.

Reply:
What exactly is the relationship between OID and code point? Or what should it be?

Reply:
IMO it should be a one-to-one relationship as per [TLS1.3](https://www.rfc-editor.org/rfc/rfc8446): If it were not, TLS clients and servers using different algorithm implementations, but the same code point would not interoperate and nobody would understand why. This of course only is a problem until an algorithm gets standardized